app/views/container_dashboard/show.html

```diff
@@ -0,0 +1,36 @@
+<div ng-app="containerDashboard" ng-controller="containerDashboardController as dashboard" class="container-fluid container-tiles-pf containers-dashboard miq-dashboard-view">
+    <div class="row row-tile-pf" >
+        <div class="col-xs-6 col-sm-4 col-md-2">
+            <div pf-aggregate-status-card status="objectStatus.providers" show-top-border="true" layout="tall"></div>
+        </div>
+        <div class="col-xs-12 col-sm-12 col-md-10">
+            <div class="row">
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.nodes" url="navigation" show-top-border="true" layout="mini"></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
+                    <div  pf-aggregate-status-card status="objectStatus.containers" url="navigation" show-top-border="true" layout="mini"></div>
+                </div>
+                <div class="col-xs-6 col-sm-6 col-md-3">
```

Comment:
if the class here repeats itself and it's long, worth to consider giving it a shorter/more readable name and add it to css